### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by orby-tech**
# Contribution checklist

## Brief description
- Added scripts for quickly setting up the application.
- Added menu to README.md
- Added Pre-requisites block to README.md

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [*] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [*] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: scripts/fast-start.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+sh ./scripts/presetup-rush.sh
+sh ./scripts/build.sh
+sh ./scripts/build.sh
```
Comment (haiodo) [inline scripts/fast-start.sh]: Why build is two times?

Comment (orby-tech) [reply scripts/fast-start.sh]: Thank u, fixed

**Review by haiodo — APPROVED** [review]

**Comment by haiodo** [comment]
Hi @orby-tech, thanks for contribution, one more small step before we could merge is to sign all commits with GPG. Thanks.

**Comment by orby-tech** [comment]
> Hi @orby-tech, thanks for contribution, one more small step before we could merge is to sign all commits with GPG. Thanks.

Great! Done

**Pull request merged**